modules/op-annotations-for-automatic-pruning-taskruns-pipelineruns.adoc

```diff
@@ -27,7 +27,7 @@ The `keep` and `keep-since` annotations are mutually exclusive. For any resource
 
 For example, consider the following annotations that retain all task runs and pipeline runs created in the last five days, and deletes the older resources: 
 
-.Example of auto-pruning annotations
```

Comment:
I think that this was actually fine as it was, since the only other use of `.Example` I really see in the docs is `.Example output`. I haven't seen `.Example:` really used elsewhere. Just a suggestion though, not a blocker.